Include/httpClient/pal.h

```diff
@@ -60,8 +60,8 @@
 
 #else
 
-#ifndef HC_ANDROID_API
-#define HC_ANDROID_API (HC_PLATFORM == HC_PLATFORM_ANDROID)
+#if !defined(HC_ANDROID_API) && HC_PLATFORM == HC_PLATFORM_ANDROID
+#define HC_ANDROID_API
 #endif
```

Comment:
Perhaps just change how this is used? instead of #ifdef HC_ANDROID_API try #if HC_ANDROID_API?

Reply:
I just looked at how HC_UWP_API worked an it works this way. I'll change to align.

---
In reply to: [212404190](https://github.com/Microsoft/libHttpClient/pull/257#discussion_r212404190) [](ancestors = 212404190)